### PR TITLE
Gather statistics for spam and virus messages

### PR DESCRIPTION
**Pull request by fyfe**
Add support for gathering statistics about spam and virus emails.

### CODE REVIEW

File: modoboa/extensions/stats/management/commands/logparser.py [hunk 1]
```diff
@@ -297,6 +297,14 @@ def _parse_line(self, line):
         if not m:
             return
         host, prog, pid, log = m.groups()
+        m = re.search("(INFECTED|SPAMMY) .* <[^>]+> -> <[^@]+@([^>]+)>.*", log)
```
Comment (tonioo) [inline modoboa/extensions/stats/management/commands/logparser.py]: And what about SPAM ?

**Comment by tonioo** [comment]
That's a great feature, thank you :)

File: modoboa/extensions/stats/management/commands/logparser.py (hunk 1 above)
Comment (fyfe) [inline modoboa/extensions/stats/management/commands/logparser.py]: The regular expression catches the type virus (INFECTED) or spam (SPAMMY) in group 1, group 2 contains the domain. The if statements on lines 303 and 305 identify it its spam or a virus and increment the relevant counter.

Comment (tonioo) [inline modoboa/extensions/stats/management/commands/logparser.py]: Sorry, my question was unclear. In addition to the keywords you already use, there is another one which is ```SPAM```. I just wonder why you don't catch it.

Comment (fyfe) [inline modoboa/extensions/stats/management/commands/logparser.py]: Because I didn't have any in my logs :)

**Pull request merged**